### ISSUE TITLE
grid with sprites v2 example column and rows swapped

### ISSUE THREAD

**Issue by jbrusey** [pythonarcade/arcade#777]
## Bug Report

If the example code is altered to have a different number of columns than rows then some grid locations don't get sprites

### Actual behavior:

Black squares for final column

### Expected behavior:

White squares in final column

### Steps to reproduce/example code:

https://arcade.academy/examples/array_backed_grid_sprites_2.html#array-backed-grid-sprites-2

alter ROW_COUNT to 4 in line 15 and COLUMN_COUNT to 5 in line 16

### What should be added/changed?

Propose changing lines 54 and 56 so that the constant values match the variable that they refer to.

**Comment by einarf** [comment]
I think it was fixed in : https://github.com/pythonarcade/arcade/issues/755

**Comment by einarf** [comment]
I'm curious if you still have this error in the development branch.

```
pip install -I https://github.com/pythonarcade/arcade/archive/development.zip
```

**Comment by einarf** [comment]
This was fixed in #794
